src/flinkSql/statementUtils.test.ts

```diff
@@ -149,6 +151,67 @@ describe("flinkSql/statementUtils.ts", function () {
     });
   });
 
+  describe("utils.refreshFlinkStatement", function () {
```

Comment:
New tests over this function in statementUtils. Was previously only tested via CCloudResourceLoader's method of the same name calling into this, hence also the compound naming scheme.

Tests include the use of `raceWithTimeout`.